android/libraries/rib-base/src/main/java/com/badoo/ribs/core/routing/activator/UnhandledChildActivator.kt

```diff
@@ -0,0 +1,17 @@
+package com.badoo.ribs.core.routing.activator
+
+import android.os.Parcelable
+import com.badoo.ribs.core.Node
+import com.badoo.ribs.core.routing.history.Routing
+import com.badoo.ribs.util.RIBs
+
+class UnhandledChildActivator<C : Parcelable> :
+    ChildActivator<C> {
+    override fun activate(routing: Routing<C>, child: Node<*>) {
+        RIBs.errorHandler.handleNonFatalError("Child requires client code activation! Routing: $routing, child: $child")
```

Comment:
routing + node without toString

Reply:
Node has ticket #187 
Routing is data class